src/axom/inlet/tests/inlet_Inlet.cpp

```diff
@@ -669,6 +669,25 @@ TEST(inlet_Inlet, mixLevelTables)
   EXPECT_EQ(intVal, 1);
 }
 
+TEST(inlet_Inlet_verify, checkVerify) {
+  std::string testString = "field1 = true; field2 = 5632; NewTable = { str = 'hello'; integer = 32 }";
+  DataStore ds;
+  auto inlet = createBasicInlet(&ds, testString);
+  std::shared_ptr<axom::inlet::Field> currField;
+  currField = inlet->addBool("NewTable/field3");
+  ds.getRoot()->getGroup("NewTable")->createViewString("required", "string");
+  EXPECT_FALSE(inlet->verify());
+
+  ds.getRoot()->getView("NewTable/required")->rename("renamedView");
+  currField = inlet->addString("NewTable/str");
+  currField->required(true);
+  currField = inlet->addInt("NewTable/int");
+  EXPECT_TRUE(inlet->verify());
+
+  ds.getRoot()->getGroup("NewTable/int")->createView("required");
```

Comment:
Do this through inlet for a non-existent value and mark it required instead of directly through the datastore to ensure that Inlet is broken/working.